### ISSUE TITLE
Discussion: Other UI Elements

### ISSUE THREAD

**Issue by mkaito** [mkaito/VikingReborn#18]
The aesthetic that VikingUI provides is quite distinct from the Carbine UI. I might be the only one that finds this jarring. Opening some windows, like the tradeskill window, which sport the default look is unpleasant to me.

I would eventually like to reskin _everything_ in a way that provides a consistent aesthetic. This is a lot of work, evidently. And even more work to keep up with patches. Ideally, we would get a few volunteers, each of which can take care of a window or three.

I'll just leave this here to provide for discussion.

**Comment by mgasparel** [comment]
Yes, this is a great idea. Maybe we can start with coming up with a style guide? This way we can get consistent colors, padding, etc... on all windows.

**Comment by mkaito** [comment]
The style guide is something that's been on my mind for a while. I have basic design notions due to my background in graphic and web design, but my colour theory and user experience knowledge are somewhat lacking. Hopefully we can work on that together. I'll start writing something and make a pull request so we can shape the guide up together.

**Comment by mkaito** [comment]
Actually, it would be great to have @itsthatguy chime in on that, since he's the original author of VikingUI. I know he's no longer interested in developing the thing, but maybe he's got something like that lying around.

**Comment by itsthatguy** [comment]
@mkaito I can get together a color palette, at the least.

**Comment by sevelev** [comment]
I can help with some of the reskinning.  I have already put up a few reskinned addons curse that are based on VikingUI color palette.  They are the "Bb_" addons.  You are welcome to use them or any part of them and change the name to Viking.

**Comment by mkaito** [comment]
Oh, I see, you're blak_bird! Welcome aboard! Those reskins of yours look nice enough. We might have to adjust some minor things to keep the aesthetic more in line with Viking.

@itsthatguy that'd be grand :smile:

**Comment by sevelev** [comment]
Totally.  Those were done rather quickly.  I haven't figured out how to make my own buttons yet, so the response selection in the dialog addon still has the default textured, glowing mouseover.  I need to make a button that is transparent when not moused over, then a nice, solid rectangle of the appropriate color when moused over.  If someone could explain how to make a button sprite or sprites with varying alphas, that would be super.

**Comment by mkaito** [comment]
@Zeirk added your Dialog reskin. I guess we can merge the rest as well and then figure out how to do buttons. Would you like to send me a PR, @sevelev?
